src/adapters/pnp/PnpAgent.c

```diff
@@ -722,7 +722,6 @@ int main(int argc, char *argv[])
     CloseAgent();
 
     StopAndDisableDaemon(OSCONFIG_PLATFORM, GetLog());
```

Comment:
>; [](http://example.com/codeflow?start=53&length=1)

nit -- why did you remove this line? Pls add it back if not big deal. Keeps the code ordered as it was